i18n/it.json

```diff
@@ -7,8 +7,8 @@
     "en": "it",
     "Paste does not exist, has expired or has been deleted.":
         "Questo messaggio non esiste, è scaduto o è stato cancellato.",
-    "%s requires php 5.3.0 or above to work. Sorry.":
-        "%s richiede PHP 5.3.0 o superiore.",
+    "%s requires php 5.4.0 or above to work. Sorry.":
+        "%s richiede php 5.4.0 o superiore per funzionare. Ci spiace.",
```

Comment:
So as [stated here](https://github.com/PrivateBin/PrivateBin/issues/201#issuecomment-288960981), please replace the PHP version with `%s`.